creator-node/src/app.js

```diff
@@ -77,7 +77,8 @@ const initializeApp = (port, serviceRegistry) => {
       // Disable default gauge counter to indicate if this middleware is running
       includeUp: false,
       // The buckets in seconds to measure requests
-      buckets: [0.2, 0.5, ...exponentialBucketsRange(1, 60, 8)],
```

Comment:
i think you accidentally removed these?

Reply:
whoops! added back in with new bucket range:
```
audius_cn_http_request_duration_seconds_bucket{le="0.2",status_code="200",method="GET",path="/health_check"} 22
audius_cn_http_request_duration_seconds_bucket{le="0.5",status_code="200",method="GET",path="/health_check"} 22
audius_cn_http_request_duration_seconds_bucket{le="1",status_code="200",method="GET",path="/health_check"} 22
audius_cn_http_request_duration_seconds_bucket{le="4",status_code="200",method="GET",path="/health_check"} 22
audius_cn_http_request_duration_seconds_bucket{le="15",status_code="200",method="GET",path="/health_check"} 22
audius_cn_http_request_duration_seconds_bucket{le="60",status_code="200",method="GET",path="/health_check"} 22
audius_cn_http_request_duration_seconds_bucket{le="+Inf",status_code="200",method="GET",path="/health_check"} 22
audius_cn_http_request_duration_seconds_sum{status_code="200",method="GET",path="/health_check"} 1.053093925
audius_cn_http_request_duration_seconds_count{status_code="200",method="GET",path="/health_check"} 22
```